### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by shaqdeff**
- Added app.rb along with respective methods.
- Added main.rb file.
- Created interactive console app.

Kindly review and provide feedback.
Regards.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  def main
+    puts 'Welcome to School Library App!'
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
```
Comment (mwanawabangona) [inline main.rb]: - [ ] `required`
- [ ] Great job so far 👏 
- [ ] when i'm trying to create a person I'm running into Error.
- [ ] please see screenshot below and see the highlighted area which exactly where the error is
image

**Review by mwanawabangona — COMMENTED** [review]
Hi @shaqdeff ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Running your project and make sure everything is working just fine

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb (hunk 1 above)
Comment (shaqdeff) [reply main.rb]: Hi @mwanawabangona,
On my side, when I create a person it doesn't give me that error. Weird. 🤔 What could be the issue?
Screenshot 2022-11-12 at 05 17 02

Comment (mwanawabangona) [reply main.rb]: @shaqdeff Thank you for reaching to me, can we please do a zoom call so we try and figure out whats wrong?

**Review by Ismailco — APPROVED** [review]
### Hi @shaqdeff, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**